Automattic/jetpack
# Remove legacy config pages & related code

Follow-up PR for cleaning up unused code after #10668

#### Changes proposed in this Pull Request:

* Remove `Jetpack::module_configuration_load()`, `Jetpack::module_configuration_head()`, `Jetpack::module_configuration_screen()`, `Jetpack::module_configuration_activation_screen()` and relevant code.
* Remove `jetpack_module_configuration_head_{module}`, `jetpack_module_configuration_load_{module}`, `display_activate_module_setting_{module}`, `jetpack_notices_update_settings`, `jetpack_module_configuration_screen_{module}` actions.
* Remove usage of `$_GET['configure']` and all relevant code.

#### Testing instructions:
Make sure that every module configuration link redirects to the desired location: (react) Jetpack Settings or wp-admin settings / other pages when applicable (for example for widgets):
* All Configure links in the legacy modules admin screen.
* Stats -> Configure link in header

#### Proposed changelog entry for your changes:
* Admin page: Remove legacy settings pages and related code
